phys2bids/physio_obj.py

```diff
@@ -296,7 +308,7 @@ def check_trigger_amount(self, chtrig=1, thr=2.5, num_timepoints_expected=0, tr=
         else:
             LGR.warning('The necessary options to find the amount of timepoints '
                         'were not provided.')
-
+        self.thr = thr
```

Comment:
Can you add this Attribute in the docstring of the class?

Reply:
This is already added to the docstring